R/install_sys_deps.R

```diff
@@ -0,0 +1,186 @@
+addToProfile = function(line, type = 'OS') {
+  paths = if (type == 'OS') c('.zshrc', '.bashrc', '.profile') else '.Rprofile'
+  paths = paste0('~/', paths)
```

Comment:
I hadn't thought about it, but having that sort of logic makes me nervous, I think we always want to write to ~/.Rprofile. I know that means that if there's a local Rprofile, then the one in the home directory will be ignored.

Reply:
After looking through again, I have another comment (this is copy/pasted from my full list):

I will say, I'm not wild about this, because we are making a system file if none exists and I don't think that's the best idea. Can't we just skip the loop iteration if the file doesn't exist? That way we aren't creating ~/.zshrc on ubuntu or ~/.bashrc on Mac unless it's already there, and we already know at least one will be there.